system/jlib/jmetrics.cpp

```diff
@@ -116,6 +116,19 @@ MetricsManager &hpccMetrics::queryMetricsManager()
 }
 
 
+MetricsManager::MetricsManager()
+{
+    try
+    {
+        nameValidator = "^[A-Za-z][A-Za-z0-9.]*[A-Za-z0-9]$";
+    }
+    catch (...)
+    {
+        throw MakeStringException(MSGAUD_operator, "Metrics manager initialized name validator regex with invalid expression");
```

Comment:
trivial: MakeStringException should be considered deprecated. use makeStringException in this case (no vargs), or makeStringExceptionV if there are vargs.